openpower/package/hostboot/hostboot.mk

```diff
@@ -6,7 +6,7 @@
 HOSTBOOT_VERSION_BRANCH_MASTER_P8 = 3707a3c6562bb54f43099f15836b79ea13659aa7
 HOSTBOOT_VERSION_BRANCH_MASTER = f5d53757fabb36a84abb75c2f97b66a7429f134b
 
-HOSTBOOT_VERSION ?= $(if $(BR2_OPENPOWER_POWER9),$(HOSTBOOT_VERSION_BRANCH_MASTER),$(HOSTBOOT_VERSION_BRANCH_MASTER_P8))
+HOSTBOOT_VERSION ?= ea1d8907e572423c5bc369593af6ca091752ecc3
```

Comment:
My guess is not. It looks like the automated thing is automated to break POWER9.